src/modules/shared/components/Navigation/Navigation.module.scss

```diff
@@ -195,3 +204,39 @@ $c-navigation-list-divider-color: #4E4E4E;
 		height: $c-navigation-height-sm - 0.1rem;
 	}
 }
+
+/**
+ * Breakpoints
+ */
+
+@include respond-to(md) {
+	.c-navigation {
+		height: $c-navigation-height-mobile;
+		padding: 0 2rem;
+
+		&__link {
+			height: $c-navigation-height-mobile - 0.1rem;
```

Comment:
`0.1rem` is een magic value, zit die ergens anders in een var die we kunnen toevoegen in de abstracts (zie `$card`) of wat compenseert dit net?

Reply:
Dat is gwn 1px dus wrs om een border ofzo te offsetten?

Reply:
Klopt, heb ik bij de andere variables bovenaan in een var gestoken